### PR TITLE
Fix 5.2 build by not including the deprecated package p.a.controlpanel

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

File: src/plone/restapi/services/controlpanels/configure.zcml [hunk 1]
```diff
@@ -2,7 +2,15 @@
     xmlns="http://namespaces.zope.org/zope"
     xmlns:plone="http://namespaces.plone.org/plone">
 
-  <include package="plone.app.controlpanel" file="permissions.zcml" />
+  <include
+    zcml:condition="have plone-52"
```
Comment (pbauer) [inline src/plone/restapi/services/controlpanels/configure.zcml]: maybe use `zcml:condition="not-installed plone.app.controlpanel"`?

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

File: src/plone/restapi/services/controlpanels/configure.zcml (hunk 1 above)
Comment (sneridagh) [reply src/plone/restapi/services/controlpanels/configure.zcml]: Indeed, better make them exclusive, if we don't we might have use cases when both are true.

**Comment by sneridagh** [comment]
@jenkins-plone-org please run jobs

**Comment by sneridagh** [comment]
@pbauer @tisto Why is the 3.7 test failing, while the others pass? and.. in RF tests?

**Comment by sneridagh** [comment]
Ok, now it is green!! :)

**Comment by tisto** [comment]
plone.restapi 3.7.5 released with the fix.

**Pull request merged**